src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -73,56 +91,176 @@
+     */
     public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_AGE = KeyFactory.fake("AREA_EFFECT_CLOUD_AGE");
 
+    /**
+     * Represents the {@link Key} for the color of a {@link AreaEffectCloud}
+     * created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#color()
+     */
     public static final Key<Value<Color>> AREA_EFFECT_CLOUD_COLOR = KeyFactory.fake("AREA_EFFECT_CLOUD_COLOR");
 
+    /**
+     * Represents the {@link Key} for the maximum age (in ticks) of a
+     * {@link AreaEffectCloud} created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#duration()
+     */
     public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_DURATION = KeyFactory.fake("AREA_EFFECT_CLOUD_DURATION");
 
+    /**
+     * Represents the {@link Key} for the amount of ticks the duration of an
+     * {@link AreaEffectCloud} is increased or reduced when it applies its
+     * effect.
+     *
+     * @see AreaEffectCloudData#durationOnUse()
+     */
     public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_DURATION_ON_USE = KeyFactory.fake("AREA_EFFECT_CLOUD_DURATION_ON_USE");
 
+    /**
+     * Represents the {@link Key} for the particle type of a
+     * {@link AreaEffectCloud} created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#particleType()
+     */
     public static final Key<Value<ParticleType>> AREA_EFFECT_CLOUD_PARTICLE_TYPE = KeyFactory.fake("AREA_EFFECT_CLOUD_PARTICLE_TYPE");
 
+    /**
+     * Represents the {@link Key} for amound the radius of a
+     * {@link AreaEffectCloud} grows or shrinks each time it
+     * applies its effect.
+     *
+     * @see AreaEffectCloudData#radius()
+     */
     public static final Key<MutableBoundedValue<Double>> AREA_EFFECT_CLOUD_RADIUS = KeyFactory.fake("AREA_EFFECT_CLOUD_RADIUS");
 
+    /**
+     * Represents the {@link Key} for the amount the radius of a
+     * {@link AreaEffectCloud} grows or shrinks per tick.
+     *
+     * @see AreaEffectCloudData#radiusOnUse()
+     */
     public static final Key<MutableBoundedValue<Double>> AREA_EFFECT_CLOUD_RADIUS_ON_USE = KeyFactory.fake("AREA_EFFECT_CLOUD_RADIUS_ON_USE");
 
+    /**
+     * Represents the {@link Key} for the age (in ticks) of a
+     * {@link AreaEffectCloud} created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#radiusPerTick()
+     */
     public static final Key<MutableBoundedValue<Double>> AREA_EFFECT_CLOUD_RADIUS_PER_TICK = KeyFactory.fake("AREA_EFFECT_CLOUD_RADIUS_PER_TICK");
 
+    /**
+     * Represents the {@link Key} for the age (in ticks) of a
+     * {@link AreaEffectCloud} created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#applicationDelay()
+     */
     public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_REAPPLICATION_DELAY = KeyFactory.fake("AREA_EFFECT_CLOUD_REAPPLICATION_DELAY");
 
+    /**
+     * Represents the {@link Key} for the age (in ticks) of a
```

Comment:
And this is a problem? I'm making a point in being explicit about the units since there seemed to be some confusion in #1269